Commenting out the optional packages
RequirePackage("arep");
RequirePackage("meataxe");
RequirePackage("nq");
RequirePackage("sisyphos");
RequirePackage("ve");

diff -ur src/gap3/lib/init.g.orig src/gap3/lib/init.g
--- src/gap3/lib/init.g.orig	2016-03-30 17:58:09.557517250 +0200
+++ src/gap3/lib/init.g	2016-03-30 17:58:23.265517363 +0200
@@ -1142,17 +1142,17 @@
 Print(" gap3-jm5 final 21Mar2016 -- see webusers.imj-prg.fr/~jmichel/gap3\n");
 Print(" Minimal distribution -- loading the following packages:\n");
 fi;
-RequirePackage("anusq");
+#RequirePackage("anusq");
 RequirePackage("arep");
 RequirePackage("autag");
 RequirePackage("cryst");
 RequirePackage("dce");
 RequirePackage("grim");
 RequirePackage("matrix");
-RequirePackage("meataxe");
+#RequirePackage("meataxe");
 RequirePackage("monoid");
-RequirePackage("nq");
+#RequirePackage("nq");
 RequirePackage("pcqa");
-RequirePackage("sisyphos");
-RequirePackage("ve");
+#RequirePackage("sisyphos");
+#RequirePackage("ve");
 RequirePackage("chevie"); # requires vkcurve, specht, algebra
